setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="ocs-ci",
-    version="4.6.0",
+    version="4.12.0",
```

Comment:
isn't it 4.13 ?

Reply:
No, we want to have this in release 4.12 branch. 4.13 we will put in your pr to default to 4.13 Vijay

Reply:
> No, we want to have this in release 4.12 branch. 4.13 we will put in your pr to default to 4.13 Vijay

got it